lib/routes/cosplaytele/popular.ts

```diff
@@ -0,0 +1,75 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { SUB_NAME_PREFIX, SUB_URL } from './const';
+import loadArticle from './article';
+
+export const route: Route = {
+    path: '/popular/:period',
+    categories: ['picture', 'popular'],
```

Comment:
```suggestion
    categories: ['picture'],
```
New route won't be popular